pysat/instruments/cosmic_gps.py

```diff
@@ -496,14 +498,15 @@ def download(date_array, tag, sat_id, data_path=None,
         # If data does not exist, will copy info not readable as tar
         fname = os.path.join(data_path,
                              'cosmic_' + sub_dir + '_' + yrdoystr + '.tar')
-        with open(fname, "wb") as local_file:
+        with Lock(fname, 'wb', pysat.file_timeout) as local_file:
```

Comment:
I think we should spell out `timeout=pysat.file_timeout` since this is a kwarg.

Reply:
Do you mean put it in the documentation?

This is also related to the feature request for a central way to store pysat related parameters so users can alter the timeout as needed.